README.md

```diff
@@ -32,22 +40,50 @@ Several parameters can be specified using environment variables:
+|-----------------------|--------------------------|-------------------------------------------|
+| `HEALTH_CHECK_PATH`   | `zookeeper-health-check` | Path to use for health check read/writes  |
+| `ZOOKEEPER_SERVERS`   |`localhost`               | Zookeeper servers                         |
+| `PROMETHEUS_ENDPOINT` | `0.0.0.0:8080`           | Endpoint for Prometheus metrics           |
+| `WATCHER_PERIOD`      |`600`                     | How often to do a read/write cycle        |
+| `WATCHER_TIMEOUT`     |`60`                      | How long to wait for message read         |
+
 ## Metrics
 
-### Kafka Watcher
+### Zookeeper Watcher
 
 | Metric                        | Type      | Description                                       |
 |-------------------------------|-----------|---------------------------------------------------|
+| `kafka_average_round_trip_time` | `gauge` | Average Round Trip Time in seconds |
 | `kafka_dropped_message_count` | `counter` | Number of messages that were dropped              |
 | `kafka_max_round_trip_time`   | `gauge`   | Maximum Round Trip Time in seconds                |
 | `kafka_min_round_trip_time`   | `gauge`   | Minimum Round Trip Time in seconds                |
 | `kafka_read_failure_count`    | `counter` | Number of failures reading messages               |
+| `kafka_running_average_round_trip_time` | `gauge` | Running Average Round Trip Time in seconds for last 5 messages |
 | `kafka_watcher_status`        | `gauge`   | Status of watcher: -1 = NOT_STARTED, 0 = OK, 1 = WARNING, 2 = ERROR |
 | `kafka_write_failure_count`   | `counter` | Number of failures writing messages               |
 
+
+### Zookeeper Watcher
+
+| Metric                            | Type      | Description                                       |
+|-----------------------------------|-----------|---------------------------------------------------|
+| `zookeeper_average_round_trip_time` | `gauge` | Average Round Trip Time in seconds |
+| `zookeeper_dropped_message_count` | `counter` | Number of messages that were dropped              |
+| `zookeeper_max_round_trip_time`   | `gauge`   | Maximum Round Trip Time in seconds                |
+| `zookeeper_min_round_trip_time`   | `gauge`   | Minimum Round Trip Time in seconds                |
+| `zookeeper_read_failure_count`    | `counter` | Number of failures reading messages               |
+| `zookeeper_running_average_round_trip_time` | `gauge` | Running Average Round Trip Time in seconds for last 5 messages |
+| `zookeeper_watcher_status`        | `gauge`   | Status of watcher: -1 = NOT_STARTED, 0 = OK, 1 = WARNING, 2 = ERROR |
+| `zookeeper_write_failure_count`   | `counter` | Number of failures writing messages               |
+
 ## Install (using Docker)
 
-See github.com/monasca/monasca-docker/kafka-watcher
+See github.com/monasca/monasca-docker/kafka-watcher and github.com/monasca/monasca-docker/zookeeper-watcher
```

Comment:
Both of this links will show 404 error on github

Reply:
Done